back/app/models/basket.rb

```diff
@@ -7,7 +7,7 @@ class Basket < ApplicationRecord
 
   validates :user, :participation_context, presence: true
   validate :in_budgeting_participation_context
-  validate :submitted_and_exceeds_limit, on: :basket_submission
+  validate :busket_submission, on: :basket_submission
```

Comment:
Not that important, but for consistency and search behavior better to fix the typo?
 b**u**sket - > b**a**sket